CONTRIBUTING.md

```diff
@@ -129,6 +129,21 @@ When you have set up your fork of a repository that you want to contribute to an
 4. **Validating and verifying the PR:** 
    - Reviewers can review the pull request on the GitHub website or fetch the PR using the _Fetch GitHub PR_ menu entry in the _Git Repositories_ view inside the Eclipse IDE.
    - Every PR is automatically verified to check that contributors have a [valid Eclipse Contributor Agreement (ECA)](#setting-up-your-eclipse-and-github-account).
+   ### What does a valid PR look like? (Checklist)
+   These are the basic criteria that need to be fulfilled for a PR to be considered valid. 
+   
+   **If you are a contributor, consider these points as prerequisites before your PR can be reviewed. Please ask for help if needed, either by making a comment on the PR or the [dev mailing list](#contact).**.
+
+   - All checks need to pass:
```

Comment:
```suggestion
   - All the following checks need to pass:
```

Reply:
Hm, actually I meant "checks" as in "the ones in the 'checks' tab" and I only intended to mention some of them as an example (the list was not meant to be full).

How about; `All checks in the "checks" tab need to pass, for example: [...]`

Reply:
FTR what @merks suggested was added in 55edb2fecea2e772411f435fa4659a6f1ab144e5.

Reply:
> `All checks need to pass.`

Done in 433a8082ad28680a027b4d69dee92d9dcf675d73



> (Of course English is not actually my native language and grammar is not my strongest skill.)

... you're closer to "native" than me X-D . Thank you for being grammar picky! :)